src/net_processing.cpp

```diff
@@ -62,11 +62,11 @@ static constexpr int32_t MAX_PEER_TX_IN_FLIGHT = 100;
 /** Maximum number of announced transactions from a peer */
 static constexpr int32_t MAX_PEER_TX_ANNOUNCEMENTS = 2 * MAX_INV_SZ;
 /** How many microseconds to delay requesting transactions from inbound peers */
-static constexpr int64_t INBOUND_PEER_TX_DELAY = 2 * 1000000; // 2 seconds
+static constexpr std::chrono::microseconds INBOUND_PEER_TX_DELAY{std::chrono::seconds{2}};
```

Comment:
is 8e5fbedf21b120d87a12bc6063454fa716fc18f8 part of a backport?

Reply:
Yes, as discussed in https://github.com/dashpay/dash/pull/3397#discussion_r404826870